### PR TITLE
Initial drawer & nav bar changes

### PR DESCRIPTION
**Pull request by dabria**
## What is this change?
In preparation for some changes to come, we are changing the layout a bit. We are removing the top bar (except on mobile) and instead having a full and mini sidebar. We are trying to use space more effectively, and we have more space horizontally to work with.

Previously the drawer would toggle open. The quick nav was always visible. Now, the drawer is always open, unless on a smaller resolution - OR if someone toggles the drawer.

### CODE REVIEW

File: src/lib/component/partial/AppBar.js [hunk 1]
```diff
@@ -70,6 +70,7 @@ class AppBar extends React.Component {
   };
 
   handleToggleDrawer = () => {
+    console.log(this.state);
```
Comment (jamesdphillips) [inline src/lib/component/partial/AppBar.js]: remove

File: src/lib/component/base/AppLayout/AppLayout.js [hunk 2]
```diff
@@ -156,19 +160,20 @@ class AppLayout extends React.PureComponent {
...
-        <div className={classes.root}>
+        <div
+          className={classnames(
+            classes.root,
+            !this.props.mobile ? classes.desktop : classes.mobile,
+          )}
+        >
           <div className={classes.topBarContainer}>
-            <ResizeObserver onResize={this.handleTopBarResize} />
+            {/* <ResizeObserver onResize={this.handleTopBarResize} /> */}
```
Comment (jamesdphillips) [inline src/lib/component/base/AppLayout/AppLayout.js]: Do we want to remove this? Looks like `topBarHeight` is still being used.

File: src/lib/component/partial/AppLayout.js [hunk 3]
```diff
@@ -32,8 +41,25 @@ class AppLayout extends React.PureComponent {
...
+    this.setState({
+      showBar: window.innerWidth <= 400,
+      fullDrawer: window.innerWidth >= 700,
+    });
+  }
+
   render() {
-    const { namespace: namespaceParam, fullWidth, children } = this.props;
+    const {
+      classes,
```
Comment (jamesdphillips) [inline src/lib/component/partial/AppLayout.js]: is this being used?

Comment (dabria) [reply src/lib/component/partial/AppLayout.js]: Nope

File: src/lib/component/base/AppLayout/AppLayout.js (hunk 2 above)
Comment (dabria) [reply src/lib/component/base/AppLayout/AppLayout.js]: Sure

**Pull request merged**